include/z64actor.h

```diff
@@ -1183,4 +1183,98 @@ typedef enum {
     /* 0xC8 */ CLEAR_TAG_SMOKE = 200
 } ClearTagType;
 
+typedef enum {
+    /* 0x00 */ TATL_HINT_ID_DEFAULT,
+    /* 0x01 */ TATL_HINT_ID_SNAPPER,
+    /* 0x02 */ TATL_HINT_ID_MINI_BABA,
+    /* 0x03 */ TATL_HINT_ID_MAD_JELLY,
+    /* 0x04 */ TATL_HINT_ID_SKULLTULA,
+    /* 0x05 */ TATL_HINT_ID_RED_CHUCHU,
+    /* 0x06 */ TATL_HINT_ID_BLUE_CHUCHU,
+    /* 0x07 */ TATL_HINT_ID_DEKU_BABA,
+    /* 0x08 */ TATL_HINT_ID_BIO_DEKU_BABA,
+    /* 0x09 */ TATL_HINT_ID_WILTED_DEKU_BABA,
+    /* 0x0A */ TATL_HINT_ID_DEXIHAND,
+    /* 0x0B */ TATL_HINT_ID_NEJIRON,
+    /* 0x0C */ TATL_HINT_ID_GIANT_BEE,
+    /* 0x0D */ TATL_HINT_ID_DODONGO,
+    /* 0x0E */ TATL_HINT_ID_DEEP_PYTHON,
+    /* 0x0F */ TATL_HINT_ID_DEATH_ARMOS,
+    /* 0x10 */ TATL_HINT_ID_DINOLFOS,
+    /* 0x11 */ TATL_HINT_ID_FIRE_KEESE,
+    /* 0x12 */ TATL_HINT_ID_KEESE,
+    /* 0x13 */ TATL_HINT_ID_ARMOS,
+    /* 0x14 */ TATL_HINT_ID_EENO,
+    /* 0x15 */ TATL_HINT_ID_DRAGONFLY,
+    /* 0x16 */ TATL_HINT_ID_IGOS_DU_IKANA,
+    /* 0x17 */ TATL_HINT_ID_GARO,
+    /* 0x18 */ TATL_HINT_ID_GARO_MASTER,
+    /* 0x19 */ TATL_HINT_ID_WART,
+    /* 0x1A */ TATL_HINT_ID_GOMESS,
+    /* 0x1B */ TATL_HINT_ID_GOHT,
+    /* 0x1C */ TATL_HINT_ID_BLUE_BUBBLE,
+    /* 0x1D */ TATL_HINT_ID_LACKEYS_DARK_ROOM,
+    /* 0x1E */ TATL_HINT_ID_LACKEYS_LIGHT_ROOM,
```

Comment:
Note: their official names, according to the Zelda Encyclopedia, are either "King's Lackeys" or "Igos du Ikana's Lackeys". I would put `KINGS_LACKEYS` here just to make it more clear what this is referring to; these guys are never named in-game, so I'd rather be slightly verbose in this instance to make it more clear to the average reader what these are.